lib/auth/index.js

```diff
@@ -71,13 +71,15 @@ const setPicture = (user, profile) => {
  */
 
 const generateStrategy = provider => {
-  // Geneate routes
+  // Generate routes
+  const options = provider === 'google' ? { scope: ['profile'] } : {};
```

Comment:
we should probably require 'email' scope in addition to profile, because we are trusting the user's email input without verification.